CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.1.0
+- Uplifted spring-boot version from 2.0.1.RELEASE to 2.7.5.
+- Uplifted spring-security-ldap version to 5.7.5 from 5.0.5.RELEASE to 5.7.5.
```

Comment:
First occurence of "to 5.7.5 " should be removed, I suppose.

Reply:
Thanks, updated the file